### PR TITLE
server: change db-table/{table_partition_id} http api

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
TiFlash maybe use tableID or paritionID to requst the `http://{IP}:10090/db-table/{table_id}` http api, if use partitionID, we should also return the partition table info.

### Check List 

Tests 

 - Unit test

Code changes

 - Has exported function/method change

### CODE REVIEW

**Comment by crazycs520** [comment]
@zanmato1984 PTAL

File: server/http_handler.go [hunk 1]
```diff
@@ -1700,18 +1700,45 @@ func (h dbTableHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
...
+		dbTblInfo.DBInfo = dbInfo
+		writeData(w, dbTblInfo)
 		return
 	}
-	dbTblInfo.TableInfo = tbl.Meta()
-	dbInfo, ok := schema.SchemaByTable(dbTblInfo.TableInfo)
-	if !ok {
-		log.Warnf("can not find the database of table id: %v, table name: %v", dbTblInfo.TableInfo.ID, dbTblInfo.TableInfo.Name)
-		writeData(w, dbTblInfo)
+	dbTblInfo.TableInfo, dbTblInfo.DBInfo = findTableByPartitionID(schema, int64(tableOrPartitionID))
```
Comment (winkyao) [inline server/http_handler.go]: Add some comments

File: server/http_handler.go [hunk 2]
```diff
@@ -1685,7 +1685,7 @@ type dbTableInfo struct {
 func (h dbTableHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	params := mux.Vars(req)
 	tableID := params[pTableID]
-	tblID, err := strconv.Atoi(tableID)
+	tableOrPartitionID, err := strconv.Atoi(tableID)
```
Comment (winkyao) [inline server/http_handler.go]: Name it physicalID?

Comment (crazycs520) [reply server/http_handler.go]: done,.

File: server/http_handler.go (hunk 1 above)
Comment (crazycs520) [reply server/http_handler.go]: done.

**Pull request merged**